### ISSUE TITLE
Context menu option not showing up on PostgreSQL tables

### ISSUE THREAD

**Issue by nphmuller** [ecirpnaes/SimpleDataScripter#4]
Extension version; `0.1.3`
Azure Data Studio version: `1.16.1`

Repro:
- Install PostgreSQL extension
- Install this extension
- Connect to PostgreSQL database
- In Servers menu (left) go to: MyPostgreServer -> Databases ->MyDatabase -> Tables -> MyTable
- Right click MyTable

Expected:
Context menu with option: `Script Table Data`

Actual:
There's no `Script Table Data` option.

**Comment by nphmuller** [comment]
The reason for the option not showing up seems to be this:
https://github.com/ecirpnaes/SimpleDataScripter/blob/5a254a1cf8467a7760e72a67a5fe4625ac8ccdf7/package.json#L36

Although scripting the table data won't of course magically work if the option shows up.

**Comment by ecirpnaes** [comment]
You are correct. The extension only works with MSSQL at this time. When I
wrote the scripting extension, the PostgresSQL extension was in
beta/preview (which is still true, I believe), so it was not sensible to
code against something that was likely to change.

**Comment by nphmuller** [comment]
What’s your take on it now? Since the extension is still in preview I don’t think it’s something you’re interested in soon right? Would it be better if I closed this issue?

**Comment by ecirpnaes** [comment]
Nah. Leave it open. It'll be a reminder to me to address the issue soon.

**Comment by machervaz** [comment]
Would be nice for us if you could do it

**Comment by daaa57150** [comment]
I was wondering why I couldn't see it...
Is it still planned ?
